### ISSUE TITLE
Data augmentation using generative adversarial networks (CycleGAN) to improve generalizability in CT segmentation tasks

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#137]
# TL;DR
A study was conducted to augment data by transforming non-contrast CT images into contrast CT images using CyCleGAN. The segmentation accuracy was achieved in the data with abundant contrast CT images, but not in hospitals with few contrast CT images. By transforming non-contrast images into contrast images and learning from them, they succeeded in greatly improving the accuracy of the latter.

![image](https://user-images.githubusercontent.com/24524018/95273859-1e784780-087f-11eb-8dc8-464787e0663a.png)

# Why it matters:

# Paper URL
https://www.nature.com/articles/s41598-019-52737-x

# Submission Dates(yyyy/mm/dd)

# Authors and institutions

Veit Sandfort, Ke Yan, Perry J. Pickhardt, Ronald M. Summers
- Imaging Biomarkers and Computer-Aided Diagnosis Laboratory
- University of Wisconsin School of Medicine and Public Health

# Methods

# Results

# Comments
